test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/ViewEngineController.ViewWithPaths.txt

```diff
@@ -0,0 +1,21 @@
+﻿<Layout>
+    /Views/ViewWithPaths/_Layout.cshtml
+    /Views/ViewWithPaths/Index.cshtml
+</Layout>
+
+<ViewStart>
+    Views\ViewWithPaths\_ViewStart.cshtml
```

Comment:
why is this case using the Windows-specific path separator?  should be consistent whatever we're rendering